pkg/kubelet/cm/devicemanager/endpoint.go

```diff
@@ -96,20 +96,20 @@ func (e *endpointImpl) callback(resourceName string, devices []pluginapi.Device)
 func (e *endpointImpl) run() {
 	stream, err := e.client.ListAndWatch(context.Background(), &pluginapi.Empty{})
 	if err != nil {
-		klog.Errorf(errListAndWatch, e.resourceName, err)
+		klog.ErrorS(err, errListAndWatch, "resource", e.resourceName)
```

Comment:
```suggestion
		klog.ErrorS(err, errListAndWatch, "resourceName", e.resourceName)
```

When passing a resource name, let's just name it `resourceName` so that if someone passes a struct they can use `resource` key.